src/Command/CleanupSnapshotsCommand.php

```diff
@@ -14,16 +14,28 @@
 namespace Sonata\PageBundle\Command;
 
 use Sonata\PageBundle\Model\SiteInterface;
+use Sonata\PageBundle\Model\SiteManagerInterface;
+use Sonata\PageBundle\Service\Contract\CleanupSnapshotBySiteInterface;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * Cleanups the deprecated snapshots.
- */
-final class CleanupSnapshotsCommand extends BaseCommand
+final class CleanupSnapshotsCommand extends Command
 {
     protected static $defaultName = 'sonata:page:cleanup-snapshots';
+    private CleanupSnapshotBySiteInterface $cleanupSnapshotBySite;
+    private SiteManagerInterface $siteManager;
+
+    public function __construct(
+        CleanupSnapshotBySiteInterface $cleanupSnapshotBySite,
+        SiteManagerInterface $siteManager
+    ) {
+        $this->cleanupSnapshotBySite = $cleanupSnapshotBySite;
+        $this->siteManager = $siteManager;
+
+        parent::__construct();
```

Comment:
Either the parent construct goes first, or last. It should be consistent across classes, at least in this project. Can you take a look at other bundles? (I think it is better first)

Reply:
I moved to the top :)